client/components/container/AppContainer.js

```diff
@@ -0,0 +1,8 @@
+import { connect } from 'react-redux';
+import App from '../presentational/App';
+
+const mapStateToProps = (state) => ({
+  isOrdersVisible: state.service.isOrdersVisible,
```

Comment:
@IgorChaikin now all data is stored in `state.service.data`, lets remove this intermediate `service.data` path. For example:

```js
state = {
  categories: [],
  items: [],
  orders: [],
  activeFilters: [],
  filters: [],
  discounts: [],
  activeCategory: ''
}
```